### PR TITLE
Training code for person-reid model 0300

### PR DESCRIPTION
**Pull request by DmitriySidnev**
@sovrasov, @snosov1, this is update for the new 0300 model with config-file and necessary code modifications.

### CODE REVIEW

File: pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml [hunk 1]
```diff
@@ -4,10 +4,11 @@ model:
   feature_dim: 512
   fpn: True
   fpn_dim: 256
+  fpn_process: 'max_pooling'
 
 data:
   type: 'image'
-  sources: ['msmt17', 'market1501']
```
Comment (AlexanderDokuchaev) [inline pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml]: Should config for 200 model be changed?
And how about update readme and add checkpoint for new model?

Comment (DmitriySidnev) [reply pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml]: `sources` in 0200 model config reverted. Readme file updated with the new 0300 model.

Comment (DmitriySidnev) [reply pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml]: May be we should delete 0200 model? New 0300 model replaces 0200 one that also is removed from upcoming OpenVINO release.

**Comment by AlexanderDokuchaev** [comment]
run tests

File: pytorch_toolkit/person_reidentification/config/default_config.py [hunk 2]
```diff
@@ -30,6 +30,7 @@ def get_default_config():
     cfg.model.feature_dim = 512
     cfg.model.fpn = True
     cfg.model.fpn_dim = 256
+    cfg.model.fpn_process = 'concatenation'
```
Comment (sovrasov) [inline pytorch_toolkit/person_reidentification/config/default_config.py]: Currently we have three FPN-related parameters there. I suggest to join them into `cfg.model.fpn` group.

File: pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml (hunk 1 above)
Comment (AlexanderDokuchaev) [reply pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml]: Yes, if 300 model replaced 200 that you can remove.

File: pytorch_toolkit/person_reidentification/config/default_config.py (hunk 2 above)
Comment (DmitriySidnev) [reply pytorch_toolkit/person_reidentification/config/default_config.py]: Done.

File: pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml (hunk 1 above)
Comment (DmitriySidnev) [reply pytorch_toolkit/person_reidentification/config/person-reidentification-retail-0200.yaml]: 0200 model removed.

**Pull request merged**